src/models/Activity.js

```diff
@@ -0,0 +1,20 @@
+const moongose = require('mongoose');
+
+const { Schema } = moongose;
+
+const activitySchema = new Schema({
+  name: {
+    type: String,
+    require: true,
+  },
+  description: {
+    type: String,
+    require: true,
+  },
+  isActive: {
+    type: Boolean,
+    require: true,
```

Comment:
I didn't set it to true by default because enabling or disabling activities is something the gym admin does. Therefore, they might want to create the activity but not enable it because it's not ready for members to sign up. That's why I ask for it as a parameter.